packages/augur-artifacts/src/addresses.json

```diff
@@ -175,19 +175,19 @@
   "TradingEscapeHatch": "0xdfef677bf5f66f3eeff481a587c04cb58e95b92a"
  },
  "12346": {
-  "Augur": "0x3730Ea866Aa1709eD4097Cb1a92F817AD2DC7c56",
-  "Universe": "0x09369758c16F85D7d91bE219Fe40048dddf0EAeC",
```

Comment:
Why did these change?

Reply:
That happened when I rebuilt for some reason. I'll revert these.